1.6/Dockerfile

```diff
@@ -1 +1,5 @@
 FROM golang:1.6
+
+RUN curl https://raw.githubusercontent.com/articulate/docker-consul-template-bootstrap/master/install.sh | bash
```

Comment:
We should separate this to two lines:

```
ADD https://raw.githubusercontent.com/articulate/docker-consul-template-bootstrap/master/install.sh /tmp/consul_template_install.sh
RUN bash /tmp/consul_template_install.sh && rm /tmp/consul_template_install.sh
```

What this does is allows docker to download the sh file, check the md5 of the content, then run it if its changed. If it has not changed, it will remain a cached layer.

The way you have it, it will be cached every single time, so by changing the file in github means it won't ever re-run, since nothing in the Dockerfile has changed.